### ISSUE TITLE
Allow opening multiple windows or tabs

### ISSUE THREAD

**Issue by egraether** [CoatiSoftware/SourcetrailBugTracker#215]

**Comment by egraether** [comment]
+1 for tabs by mail

**Comment by egraether** [comment]
+ 1 by mail
Windows should be closed separetely (Cmd + W on Mac) without closing the main process.
When all windows are closed there should be a default screen.

**Comment by egraether** [comment]
+1 by mail

**Comment by egraether** [comment]
+1 via #472

**Comment by egraether** [comment]
+1 via mail

**Comment by phonetagger** [comment]
How can I vote this feature request up? BTW if I had to choose between "open additional window" and tabs, I'd choose "open additional window". But both options would be nice as well. And being able to pop the tab out into a separate window, or combine a separate window back into another one as a tab. I agree that the ability to close one window without closing the other ones is pretty important. I think it would be nice if none of the windows was a "master" window and they all had the same functionality. Closing the last remaining window would close the application.
